test/integration/utils.js

```diff
@@ -0,0 +1,53 @@
+import path from 'path';
+
+import { BrowserWindow } from 'electron';
+
+const UI_DIR = path.join(__dirname, '..', '..', 'lib', 'ui');
+const BROWSER_CHROME_URL = fileUrl(path.join(UI_DIR, 'browser-modern', 'browser.html'));
+
+function fileUrl(str) {
+  let pathName = path.resolve(str).replace(/\\/g, '/');
```

Comment:
A version of this function is already defined in app/shared/paths-util. Unless I'm missing something, you should import that.